//package candy.protocol;
//好友信息

message FriendInfoLite {
    required string uid = 1;                //uid
    required string nickName = 2;           //用户昵称
    required uint32 level = 3;              //等级
    required string openid = 7;
    required string headurl = 8;
    optional int32  friendtype = 9 [default = 0]; // 0:平台好友 1：战友 2：黑名单 3：系统 4:最近联系人 5:最近战斗的人
    optional bool   isYellowDmd     = 11 [default = false];  //是否是黄钻
    optional bool   isYellowDmdYear = 12 [default = false];  //是否是年费黄钻
    optional int32  YellowDmdLvl    = 13 [default = 0];      //黄钻等级
    optional int64	lastLogTime		= 23 [default = -1];	//最后一次登陆或退出游戏的时间(秒)
    optional string lastLogTimeStr  = 24 [default =""];     //
    optional int32  QPCount         = 25 ;
    repeated int32  lvlscores       = 26 ;
    optional string jsonstr         = 14 [default = ""];
    optional int32  dallytotalsocre = 15 [default =0];
    optional int32  update_day      = 16 [default =0];
    repeated int32  dallylvlscores  = 17 ;
    optional int32  total_stars     = 18 ;

}

